File: Desenvolvimento/client/src/@fuse/types/models/tramite-check-list.ts
import { FluxoItemCheckList } from './fluxo-item-checklist';
import { BaseModel } from '../base-model';
import { Tramite } from './tramite';

export class TramiteChecklist extends BaseModel {
    constructor(
        public id: number,
        public checado: boolean,
        public tramiteId?: number,
        public fluxoItemChecklistId?: number,
        public tramite?: Tramite,
        public fluxoItemChecklist?: FluxoItemCheckList ,
    ) { super(); }
}

File: Desenvolvimento/server/src/ApplicationCore/Entities/TramiteChecklist.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class TramiteChecklist : IBaseEntity<long>
    {
        public long Id { get; set; }
        public bool Checado { get; set; }
        public long FluxoItemChecklistId { get; set; }
        public long TramiteId { get; set; }
        public Tramite Tramite { get; set; }
        public FluxoItemChecklist FluxoItemChecklist { get; set; }
    }
}
